README.md

```diff
@@ -63,18 +63,18 @@ and add the appropriate language extension.
 
 The available languages, in alphabetical order, with their artifact id are:
 
-| Language | Artifact Id        |
-|:--------:|:------------------:|
-| Atom     | `polyglot-atom`    |
-| Groovy   | `polyglot-groovy`  |
-| Clojure  | `polyglot-clojure` |
-| Ruby     | `polyglot-ruby`    |
-| Scala    | `polyglot-scala`   |
-| YAML     | `polyglot-yaml`    |
+| Language | Artifact Id        | Language reference |
+|:--------:|:------------------:|:------------------:|
+| Atom     | `polyglot-atom`    | [atom](http://hackage.haskell.org/package/atom |
+  Groovy   | `polyglot-groovy`  |                    |
+| Clojure  | `polyglot-clojure` |                    |
+| Ruby     | `polyglot-ruby`    |                    |
+| Scala    | `polyglot-scala`   |                    |
+| YAML     | `polyglot-yaml`    | [yaml.org](http://yaml.org/) |
 
```

Comment:
Well, atom is new for me.
We can't tell that every one is to know all languages